pkg/server/server.go

```diff
@@ -122,6 +122,10 @@ func New(sessionStore session.Store, runConfig config.RuntimeConfig, teams map[s
 
 	group.GET("/desktop/token", s.getDesktopToken)
 
+	// Resume to start an OAuth flow
+	group.POST("/:id/resumeStartOauth", s.resumeStartOauth)
+	group.POST("/resumeCodeReceivedOauth", s.resumeCodeReceivedOauth)
```

Comment:
These should be 

```suggestion
	group.POST("/sessions/:id/resumeStartOauth", s.resumeStartOauth)
	group.POST("/sessions/:id/resumeCodeReceivedOauth", s.resumeCodeReceivedOauth)
```

Why doesn't the second one have a session id?

Reply:
Gotcha!